tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -97,3 +99,17 @@ def cleanup(self):
 
     def executor_browser(self):
         return ExecutorBrowser, {"webdriver_url": self.server.url}
+
+
+def run_info_extras(**kwargs):
+    osReleaseCommand = "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').ReleaseId"
```

Comment:
```
  /home/test/web-platform-tests/tools/wptrunner/wptrunner/browsers/edge.py:105: DeprecationWarning: invalid escape sequence \S
```

I guess this should have used double backslashes.

Reply:
Should probably be a raw string.

Reply:
Do I need to make a fresh PR to fix this up?

Reply:
If you could, yes, please.

Reply:
I've put up a CR over [here ](https://github.com/web-platform-tests/wpt/pull/16234)